CHANGELOG.md

```diff
@@ -6,38 +6,40 @@ _For older release notes, see_ [PWA Studio releases][].
+-  **Added PWA Studio metapackages** — In this release, we introduce our PWA Studio metapackages: One for [Magento Open Source](https://github.com/magento/magento2-pwa) and one for [Adobe Commerce](https://github.com/magento/magento2-pwa-commerce). These metapackages provide a new way to add any additional Open Source or Commerce features needed by your PWA modules. The PWA Studio team will also use these metapackages to add new features to the Open Source and Commerce code bases as needed. In fact, we did that in this release. We added GraphQL endpoints and fields to the metapackages to fix some `Cart`-related bugs. GitHub PRs: [4][], [1][].
 
     **UPDATES REQUIRED!** This release and all future PWA Studio releases will have dependencies on these metapackages. That means you need to add one or both of the metapackages to your PWA apps (depending on your backend target). Instructions for installing these packages are in the README files for each metapackage repo:
 
-    **For Open Source backends**: Install the [Magento Open Source metapackage](https://github.com/magento-commerce/magento2-pwa).
+    **For Open Source backends**: Install the [Magento Open Source metapackage](https://github.com/magento/magento2-pwa).
 
-    **For Adobe Commerce backends**: Install the [Adobe Commerce metapackage](https://github.com/magento-commerce/magento2-pwa-commerce).
+    **For Adobe Commerce backends**: Install the [Adobe Commerce metapackage](https://github.com/magento/magento2-pwa-commerce).
 
 -  **Added new PWA Tailwind theming to Venia Header** — Refactored the Venia site `Header` component to use our new [Tailwind](https://tailwindcss.com/) theming framework. The `Header` component is the first component to use our theming framework. Other components will follow in the coming releases. GitHub PR: [3472][].
 
 -  **Added Venia Sample Data metapackage** — This metapackage provides a new way to create your own sample data for your Venia-based PWA sites. For more details, see the [Venia Sample Data metapackage repo](https://github.com/magento-commerce/venia-sample-data-modules). GitHub PRs: [3473][], [2][].
 
--  **Added new GraphQL endpoint for Contact and Newsletter forms** — You can now use GraphQL to submit Contact Us and Newsletter form data to the Commerce backend. From there, you can use it to send emails and personalize other customer interactions. GitHub PR: [5][].
+-  **Added a GraphQL mutation for submitting the Contact Us form.** — You can now use GraphQL to submit the Contact Us form data to the Open Source or Commerce backend. From there, you can use it to send emails and personalize other customer interactions.
+
```

Comment:
Are you sure about the last sentence? The mutation just sends a Contact message to the merchant. Although I'm sure you can style the form any way you want, I don't foresee the ability to do anything else with it. If you want to develop a totally different form to contact the merchant, you'd have to develop a custom module.

Reply:
Yep, you're right. This was just a "see what kind of things you can do with this" sentence, which could be a variety of things. Good call.